.golangci.yml

```diff
@@ -109,7 +110,10 @@ linters-settings:
     simple: true
     range-loops: true # Report preallocation suggestions on range loops, true by default
     for-loops: false # Report preallocation suggestions on for loops, false by default
-
+  gocritic:
+    disabled-checks:
+    # Disabling appendAssign b/c it reports false positives
```

Comment:
Are there many false positives?

Reply:
I reports 1 false positive.  

cmd/storj-sim/prefix.go:87:12: appendAssign: append result not assigned to the same slice (gocritic)
        buffer = append(writer.buffer, data...)
                 ^
make: *** [lint] Error 1

The assignment appears to be intentional b/c the assignment to write.buffer is deferred in the next line.

Reply:
Note you can skip specific lines with an appropriate comment rather than disable the whole check.

Reply:
Thanks for the tip.  I've add //nolint to the line and re-enabled the appendAssign check